pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -363,8 +364,8 @@ func (r *PyTorchJobReconciler) UpdateJobStatus(job interface{},
 
 	logger := commonutil.LoggerForJob(pytorchjob)
 
-	// Set StartTime.
-	if jobStatus.StartTime == nil {
+	if !trainutil.IsJobSuspended(&pytorchjob.Spec.RunPolicy) && jobStatus.StartTime == nil {
```

Comment:
It is slight different than what we do in Job & MPIJob, where we don't set StartTime to nil, just update the StartTime on resume or first run. Any reason to implement it differently here, maybe we could follow the same pattern?

Reply:
That makes sense.

Reply:
Done.